Source/plugins/Types.h

```diff
@@ -267,13 +271,13 @@ namespace RPC {
         {
             ASSERT(_controller == nullptr);
 
+            _controller = Connector::Connector::Instance().Controller();
```

Comment:
Use the C function i.s.o. the C++ functionality, so remark for this class declaration.

Reply:
Done